File: labServiceUIJava/labServiceUIJava/src/main/java/com/example/labserviceuijava/service/ServiceArtist.java
package com.example.labserviceuijava.service;

import com.example.labserviceuijava.model.Artist;
import com.example.labserviceuijava.repository.ArtistDBRepository;

import com.example.labserviceuijava.repository.exceptions.RepositoryException;
import com.example.labserviceuijava.validator.ArtistValidator;

import java.security.Provider;

public class ServiceArtist {

    private final ArtistDBRepository artistDBRepository;

    private final ArtistValidator artistValidator;

    public ServiceArtist(ArtistDBRepository artistDBRepository, ArtistValidator artistValidator) {
        this.artistDBRepository = artistDBRepository;
        this.artistValidator = artistValidator;
    }

    public Artist findOne(Integer id){
        return artistDBRepository.findOne(id);
    }

    public Iterable<Artist> findAll(){
        return artistDBRepository.findAll();
    }
}

File: labServiceUIcs/WindowsFormsApp2/service/ServiceArtist.cs
using System.Collections.Generic;
using WindowsFormsApp2.model;
using WindowsFormsApp2.repository;

namespace WindowsFormsApp2
{
    public class ServiceArtist
    {
        private ArtistsDBRepository artistsDbRepository;

        public ServiceArtist(ArtistsDBRepository artistsDbRepository)
        {
            this.artistsDbRepository = artistsDbRepository;
        }

        public Artist findOne(int id)
        {
            return artistsDbRepository.FindOne(id);
        }

        public IEnumerable<Artist> FindAll()
        {
            return artistsDbRepository.FindAll();
        }
    }
}